akka-cluster/src/main/scala/akka/cluster/sbr/DowningStrategy.scala

```diff
@@ -205,26 +205,12 @@ import akka.coordination.lease.scaladsl.Lease
     _allMembers.exists(m => m.uniqueAddress == node && m.dataCenter == selfDc)
   }
 
-  /**
-   * @return true if it was changed
-   */
-  private[sbr] def setReachability(r: Reachability): Boolean = {
+  private[sbr] def setReachability(r: Reachability): Unit = {
```

Comment:
I searched for `setReachability` in Cinnamon and it was not used so should be ok to change the signature here.